### ISSUE TITLE
EnergyPlus workflow fails with incorrect version

### ISSUE THREAD

**Issue by JasonGlazer** [NREL/EP-Launch#112]
*** STARTING WORKFLOW ***
Workflow FAILED: Incorrect Version found 9.0: 5ZoneEndUses.idf!
*** WORKFLOW FINISHED ***

**Comment by Myoldmopar** [comment]
So this is because the E+ workflow has a trap for version > 8.3, using the full version numeric representation 80300.  This will require a change in the energyplus workflow over in E+ land.  We can provide update workflow files for the user to drop into their E+ installation if that's sufficient.  

I guess there's one alternative, and that's to go **back** to using a full 5 (6) digit representation of the version, but always use zeroes as the patch number, regardless of the actual patch number of E+.  It will require reverting some stuff I did yesterday, but it would allow this to work.  And maybe everything else.  I'll try that.

**Comment by Myoldmopar** [comment]
(For the record, I don't think we need that version trap in the e+ workflow anyway.  If someone is taking our workflows and back-porting them to older versions of e+, we can't be responsible for what all they run into.  I will take that particular check out of there if you're ok with it.)

**Comment by Myoldmopar** [comment]
OK, this is fixed again.  I put everything back to a MMmmPP version string, but I *always* assign zero as the patch number.  I was able to test the transition stuff and it works, and now the energyplus workflow works as it sits in the 9.0.x installers.  I would still like to clean that stuff up a bit, but it's OK for now.  I pushed my branch.  I'd like for you to test it out if you get a chance.  If CI is still happy with it, I may end up merging it because I feel pretty good about it.
